### PR TITLE
update the appeals api docs

### PR DESCRIPTION
**Pull request by charleystran**
## Description of change
This PR updates the Appeals documentation per jira ticket api-1610.

### CODE REVIEW

File: modules/appeals_api/app/swagger/v0/appeals.yml [hunk 1]
```diff
@@ -3,38 +3,33 @@ info:
...
+    ### Authorization and Access
 
-    API requests are authorized through a symmetric API token, provided in an HTTP header with name "apikey".
+    To gain access to the Appeals Status API you must [request an API Key](/apply). API requests are authorized through a symmetric API token which is provided in an HTTP header named `apikey`.
 
-    ### Status Request
+    ### Data refresh
 
-    Allows a client to check benefit appeals status
+    The Lighthouse Sandbox environment points to the Caseflow Demo environment which refreshes daily. The data in this environment is mock data and does not contain any PII.
```
Comment (edmangimelli) [inline modules/appeals_api/app/swagger/v0/appeals.yml:23]: Are we wired up to caseflowdemo ?

Comment (charleystran) [reply modules/appeals_api/app/swagger/v0/appeals.yml:23]: The https://docs.google.com/document/d/1D44p9lDLqhsm8KdAIeXLIr4ylwVbjRqhEHxH_vb3r-4/edit# specifies that, but we should check what Darcy means exactly.

Comment (charleystran) [reply modules/appeals_api/app/swagger/v0/appeals.yml:23]: updated to reference UAT

File: modules/appeals_api/app/swagger/appeals_api/v1/api_description.md [hunk 2]
```diff
@@ -1,11 +1,14 @@
...
-## Technical Summary
+Because this application is designed to allow third-parties to request information on behalf of a Veteran, we are not using VA Authentication Federation Infrastructure (VAAFI) headers or Single Sign On External (SSOe).
 
-### Authorization
+### Authorization and Access
+To gain access to the decision reviews API you must [request an API Key](/apply). API requests are authorized through a symmetric API token which is provided in an HTTP header named `apikey`.
 
-API requests are authorized through a symmetric API token, provided in an HTTP header with name `apikey`.
+### Data refresh
+The Lighthouse Sandbox environment points to the Caseflow UAT environment which refreshes daily. The data in this environment is mock data and does not contain any PII.
```
Comment (edmangimelli) [inline modules/appeals_api/app/swagger/appeals_api/v1/api_description.md:14]: UAT doesn't refresh daily
https://dsva.slack.com/archives/C2ZAMLK88/p1596053539041000?thread_ts=1596053120.040900&cid=C2ZAMLK88

File: modules/appeals_api/app/swagger/v0/appeals.yml [hunk 3]
```diff
@@ -3,38 +3,33 @@ info:
...
+    ### Authorization and Access
 
-    API requests are authorized through a symmetric API token, provided in an HTTP header with name "apikey".
+    To gain access to the Appeals Status API you must [request an API Key](/apply). API requests are authorized through a symmetric API token which is provided in an HTTP header named `apikey`.
 
-    ### Status Request
+    ### Data refresh
 
-    Allows a client to check benefit appeals status
+    The Lighthouse Sandbox environment points to the Caseflow UAT environment which refreshes daily. The data in this environment is mock data and does not contain any PII.
```
Comment (edmangimelli) [inline modules/appeals_api/app/swagger/v0/appeals.yml:23]: UAT doesn't refresh daily
https://dsva.slack.com/archives/C2ZAMLK88/p1596053539041000?thread_ts=1596053120.040900&cid=C2ZAMLK88

File: modules/appeals_api/app/swagger/appeals_api/v1/api_description.md (hunk 2 above)
Comment (edmangimelli) [reply modules/appeals_api/app/swagger/appeals_api/v1/api_description.md:14]: confirmed that UAT's DB is never reset
https://dsva.slack.com/archives/C2ZAMLK88/p1596123119046300?thread_ts=1596053120.040900&cid=C2ZAMLK88

File: modules/appeals_api/app/swagger/v0/appeals.yml (hunk 3 above)
Comment (edmangimelli) [reply modules/appeals_api/app/swagger/v0/appeals.yml:23]: confirmed that UAT's DB is never reset
https://dsva.slack.com/archives/C2ZAMLK88/p1596123119046300?thread_ts=1596053120.040900&cid=C2ZAMLK88

**Pull request merged**